public/scss/base/_variables.scss

```diff
@@ -50,9 +50,9 @@ $grid-full: minmax(#{$grid-gap}, 1fr);
 $grid-content: min(48rem, 100% - #{$grid-gap} * 2);
 $grid-popout: minmax(0, 2rem);
 $grid-feature: minmax(0, 11.375rem);
-$grid-sidebar: clamp(5rem, 33vw, 21.5rem);
+$grid-sidebar: clamp(5rem, 33vw, 25rem);
 $grid-sidebar-gap: clamp(2rem, 5vw, 5.3125rem);
-$grid-sidebar-content: minmax(auto, 48rem);
+$grid-sidebar-content: min(48rem, 100% - #{$grid-sidebar} - #{$grid-sidebar-gap} * 2);
```

Comment:
Accidentally didn't account for the sidebar and gap causing huge amounts of X overflow on small screens 😅